zio-http/src/test/scala/zio/http/service/StaticServerSpec.scala

```diff
@@ -68,29 +68,29 @@ object StaticServerSpec extends HttpRunnableSpec {
     },
   )
 
-  def serverStartSpec = suite("ServerStartSpec")(
-    test("desired port") {
-      val port = 8088
-      ZIO.scoped {
-        (Server.port(port) ++ Server.app(Http.empty)).make.flatMap { start =>
-          assertZIO(ZIO.attempt(start.port))(equalTo(port))
-        }
-      }
-    },
-    test("available port") {
-      ZIO.scoped {
-        (Server.port(0) ++ Server.app(Http.empty)).make.flatMap { start =>
-          assertZIO(ZIO.attempt(start.port))(not(equalTo(0)))
-        }
-      }
-    },
-  )
+//  def serverStartSpec = suite("ServerStartSpec")(
+//    test("desired port") {
+//      val port = 8088
+//      ZIO.scoped {
+//        (Server.port(port) ++ Server.app(Http.empty)).make.flatMap { start =>
+//          assertZIO(ZIO.attempt(start.port))(equalTo(port))
+//        }
```

Comment:
Disabled test

Reply:
just temporary, to move them out in their own suite as requires different configs to run